app/views/categories/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= render partial: 'shared/nav', locals: {title:'NEW CATEGORY', symbol:'←', url: :back}%>
+
+<%= render "form", category: @category %>
+
+<br>
+
+<div>
+  <%= link_to "Back to categories", categories_path %>
+</div>
```

Comment:
- [ ] Please make sure when the user clicks the "Save" button to create the new category and user should be redirected to the home page(categories page) on success. Currently, it's redirected to the transactions page of that category.

![image](https://github.com/quavo19/budget-app/assets/83803414/4132fe45-c8fa-44e6-9669-29326ac88a7a)